doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,139 @@
+.. _gh_labels:
+
+Labeling issues and pull requests in GitHub
+###########################################
+
+The project uses GitHub issues and pull requests (PRs) to track and manage
+daily and long-term work and contributions to the Zephyr project. We use
+GitHub **labels** to classify and organize these issues and PRs by area, type,
+priority, and more, making it easier to find and report on relevant items.
+
+All GitHub issues or pull requests must be appropriately labeled.
+Issues and PRs often have multiple labels assigned,
+to help classify them in the different available categories.
+When reviewing a PR, if it has missing or incorrect labels, maintainers shall
+fix it.
+
+This saves us all time when searching, reduces the chances of the PR or issue
+being forgotten, speeds up reviewing, avoids duplicate issue reports, etc.
+
+These are the labels we currently have, grouped by type:
+
+Area:
+*****
+
+=============  ===============================================================
+Labels         ``Area:*``
+Applicable to  PRs  and issues
+Description    Indicates subsystems (e.g., Kernel, I2C, Memory Management),
+               project functions (e.g., Debugging, Documentation, Process),
+               or other categories (e.g., Coding Style, MISRA-C)  affected by
+               the bug or pull request.
+=============  ===============================================================
+
+An area maintainer should be able to filter by an area label and
+find all issues and PRs which relate to that area.
+
+Platform
+*********
+
+=============  ===============================================================
+Labels         ``Platform:*``
+Applicable to  PRs  and issues
+Description    An issue or PR which affects only a particular platform
+=============  ===============================================================
+
+To be discussed in a meeting
+****************************
+
+=============  ===============================================================
+Labels         ``API``, ``dev-review``, ``TSC``
+Applicable to  PRs  and issues
+Description    The issue is to be discussed in the following
+               `API/dev-review/TSC meeting`_ if time permits
+=============  ===============================================================
+
+.. _`API/dev-review/TSC meeting`: https://github.com/zephyrproject-rtos/zephyr/wiki/Zephyr-Committee-and-Working-Group-Meetings
+
+Minimum PR review time
+**********************
+
+=============  ===============================================================
+Labels         ``Hot Fix``, ``Trivial PR``, ``Maintainer``,
+               ``Security Review``, ``TSC``
```

Comment:
Is this the same ``TSC`` mentioned above?  If it's different, then a prefix would clarify :)

Reply:
Today it is the same, I would be happy to split it in two, but Anas was not too keen on it.